netsim/devices/eos.yml

```diff
@@ -112,3 +115,8 @@ external:
   image: none
 graphite.icon: switch
 bgp._cprop_order: [ standard, extended, large, link-bandwidth ]
+
+# Define custom device-specific attributes
+attributes:
+ node:
+  ceos.serialnumber: int  # Referenced in ceos-config.j2 template
```

Comment:
It would be nice to have attributes that match device types to minimize confusion. From that perspective, it could be either `eos.serialnumber` (or is it `serial_number` 😉) or `clab.eos.serialnumber` if we want to emphasize that it only works on containers.

Reply:
Opting for ```eos.serialnumber```, we might one day be able to apply the same setting for other providers too. The modeling of the property and the user intent remains the same